api/WritableStreamDefaultWriter.json

```diff
@@ -79,7 +79,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "preview"
+              "version_added": "14.1"
```

Comment:
Hmm, in Safari 14.1, trying to run `new WritableStreamDefaultWriter()` throws a TypeError: `WritableStreamDefaultWriter constructor takes a WritableStream`.  This error seems to imply that the constructor is supported.

Reply:
Is it behaving differently starting with 16.4?
I mean throwing an error is not quite support, I guess?

Reply:
No, it's actually the same error message in 16.4.  If the constructor wasn't supported, then we should be seeing a message along the lines of "invalid constructor", rather than "invalid arguments".

(The collector's constructor test function runs the constructor without any arguments and checks the error message.  If there was no error, or the error message refers to issues with the passed arguments, then it marks the constructor as supported.  If the error message says that the variable is not defined or isn't a constructor, it marks the constructor as not supported.)

Reply:
Oh never mind, I looked at the wrong thing in https://github.com/WebKit/WebKit/pull/7019/files#diff-60ced0e5aaa6a7542c5fdea857994da2f51781941673c114f5ece9c3a372881e. These classes have way too similar names!

Reply:
Yes, you are right and 14.1 seems likely then.